tms/views/tms_order.xml

```diff
@@ -96,12 +94,12 @@
                     <widget
                         name="web_ribbon"
                         title="Completed"
-                        invisible="stage != 'Completed'"
+                        invisible="stage_id != 3"
```

Comment:
Even so, it doesn't solve the problem. Neither by name, ID, nor reference. I don't see any valid reason to force the use of certain stages. Perhaps it would be better to manage statuses, which could be calculated based on the parameters of the stages and related factors. This approach would also enable the implementation of a module like tms_order_tier_validation, for example.